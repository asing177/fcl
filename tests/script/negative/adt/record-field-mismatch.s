type T {
  C1(int  f1, bool f2, text f3);
  C2(bool f1, text f2);
  C3(text f1);
}

global bool x;

@initial
go(T t) {
  x = t.f2;
  terminate()
}
