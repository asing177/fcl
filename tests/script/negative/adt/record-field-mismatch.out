Error at No location info:
   Conflicting fields in a record declaration:
      'int f1' at Line 2, Column 13
      'bool f1' at Line 3, Column 13
      'text f1' at Line 4, Column 13
   Record fields of the same name must have the same type.